### ISSUE TITLE
Ubuntu 20.04 is distributing a broken development build

### ISSUE THREAD

**Issue by alexandervdm** [alexandervdm/gummi#141]
Please see https://bugs.launchpad.net/ubuntu/+source/gummi/+bug/1872667 for more information. 

A repository is available for several Ubuntu targets that offers the latest stable releases at: https://packages.gummi.app/

**Comment by stockphrase** [comment]
Wrong link, friend.

**Comment by alexandervdm** [comment]
Whoops, lost a char - thanks.

**Comment by sil2100** [comment]
As mentioned in the attached bug, we will gladly help out in getting the gummi package version updated to the right version. I would be grateful for some additional info as per my latest comment on the bug.
Just so that it's also clear: this development build has been auto-synced from Debian testing, so it's not something we pulled in deliberately. The original bug was filled right before final freeze and was only brought to our attention after the 20.04 release, making removing packages completely from the release pocket 'tricky' to say the least (and not really feasible). This is why we recommended the option of backporting the 0.8.1 version as an SRU.
